src/msix/PAL/Crypto/OpenSSL/SignatureCreator.cpp

```diff
@@ -35,7 +36,7 @@ namespace MSIX
                     unique_BIO certBIO{ BIO_new_mem_buf(reinterpret_cast<void*>(certBytes.data()), static_cast<int>(certBytes.size())) };
                     unique_PKCS12 cert{ d2i_PKCS12_bio(certBIO.get(), nullptr) };
 
-                    ParsePKCS12(cert.get(), nullptr, "Unable to open PFX file");
+                    ParsePKCS12(cert.get(), pass, "Unable to open PFX file - incorrect password?");

```

Comment:
`ThrowOpenSSLErrIfFailedMsg` should be extracting useful information from OpenSSL about the nature of the error.

Reply:
I've changed this to not add the extra text, and instead leave it to the OpenSSL message.